azure-pipelines/official.yml

```diff
@@ -42,7 +24,7 @@ parameters:
 - name: EnableAPIScan
   displayName: Include APIScan with compliance tools
   type: boolean
-  default: false # enable in individual repos only AFTER updating TSAOptions.json with your own values
+  default: true
```

Comment:
Why change this? Nothing else in the template enforces that TSAOptions.json has been initialized, so even if APIScan were required, isn't it better to default to off so that users come across this comment while enabling it and realize they must also update TSAOptions.json?

Reply:
This was a mistake. I was following what I did for the Gladstone repo and I didn't realize this had its own special explanation